webwidget/src/main/java/com/fitanalytics/webwidget/FITAPurchaseReporter.java

```diff
@@ -27,6 +27,8 @@ public class FITAPurchaseReporter extends AsyncTask<FITAPurchaseReport, Integer,
 
 
     private static final String FITA_WIDGET = "fitaWidget";
+    // TODO:
+    // since we swithced to "//widget.fitanalytics.com/report_purchase.js", shouldn't we update the code here?
```

Comment:
The reporting here doesn't use the reporting script at all and instead it sends HTTP requests directly to collector.